### ISSUE TITLE
Feature Request | Custom output loader to load to graph databases...

### ISSUE THREAD

**Issue by rajib76** [ucbepic/docetl#104]
I wanted to request for this feature, where after extracting the information from the unstructured data, can we have a output loader, that can load the output to different types of database for example a graph DB, this will be useful for GraphRAG use cases.

**Comment by shreyashankar** [comment]
This would be cool! Do you imagine it having a similar interface to the custom input data parsers?

**Comment by rajib76** [comment]
Yes Shreya, I should be able to tell it to load it to a graph database or a relational db. for example if I extract some structured  information from a knowledge source, I should be able to load those information to a RDBMS
